test/e2e-go/cli/goal/expect/corsTest.exp

```diff
@@ -31,6 +31,10 @@ if { [catch {
     set ALGOD_NET_ADDRESS [::AlgorandGoal::GetAlgodNetworkAddress $TEST_PRIMARY_NODE_DIR]
     ::AlgorandGoal::CheckNetworkAddressForCors $ALGOD_NET_ADDRESS
 
+    # Hit algod with a private network access preflight request and look for 200 OK
```

Comment:
should we have a similar test that ensures the config worked as expected? ..Probably okay to rely on our unit testing for that.

Reply:
I'll look at adding additional go tests for algod and kmd, similar to what I did for [Indexer](https://github.com/algorand/indexer/pull/1620).